utils/python/CIME/XML/testlist.py

```diff
@@ -0,0 +1,85 @@
+"""
+Interface to the config_files.xml file.  This class inherits from EntryID.py
+"""
+from standard_module_setup import *
+
+from generic_xml import GenericXML
+from CIME.utils import expect, get_cime_root, get_model
+
```

Comment:
I'm not going to look too closely at this file since I don't think ACME will ever get it's tests this way.